istioctl/cmd/workload.go

```diff
@@ -205,7 +208,7 @@ Configure requires either the WorkloadGroup artifact path or its location on the
 			return nil
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
-			kubeClient, err := kubeClientWithRevision(opts.Revision)
+			kubeClient, err := kubeClientWithRevision(ctx, opts.Revision)
```

Comment:
ideally we stop using kubeClientWithRevision. Can be followup though

Reply:
Yes, I was thinking the same. However, some tests are coupled with the func, so I think I'm going to address this in the followup PRs.